weather/models.py

```diff
@@ -0,0 +1,38 @@
+from sqlalchemy import (
+    Column,
+    Integer,
+    String,
+    ForeignKey,
+    DateTime,
+    Float,
+)
+from sqlalchemy.orm import relationship
+
+from database import Base
+
+
+class City(Base):
+    __tablename__ = "cities"
+
+    id = Column(Integer, primary_key=True, index=True)
+    name = Column(String(63), nullable=False, unique=True)
+    additional_info = Column(String(255), )
+
+    temperatures = relationship(
+        "Temperature",
+        back_populates="city",
+    )
+
+
+class Temperature(Base):
+    __tablename__ = "temperatures"
+
+    id = Column(Integer, primary_key=True, index=True)
```

Comment:
The `id` column is indexed by default as a primary key. The additional `index=True` is not necessary but does not cause harm. Consider removing it for simplicity.